ambry-api/src/main/java/com/github/ambry/rest/RestUtils.java

```diff
@@ -376,6 +375,10 @@ public static final class InternalKeys {
      */
     public static final String TARGET_DATASET_KEY = KEY_PREFIX + "target-dataset";
 
+    /**
+     * The key for the target dataset version indicated by the request.
+     */
+    public static final String TARGET_DATASET_VERSION_KEY = KEY_PREFIX + "target-dataset-version-key";
```

Comment:
nit: we can remove -key suffix.

Reply:
we don't have -key suffix for internal keys.

Reply:
sorry, didn't find that before.